### ISSUE TITLE
Unhandled Promise Rejection: TypeError: Cannot read property 'pid' of undefined

### ISSUE THREAD

**Issue by lrettig** [spacemeshos/smapp#506]
**Describe the bug**

Getting this error every time I exit the app

**To Reproduce**
1. Open app
1. It says "Waiting for smesher response..." for a very long time (hours), despite the node being fully synced (#501)
1. Click the "X" to close the app - no prompt (#502), it immediately begins to shut down, closes the underlying node, but then hangs

**Expected behavior**
Expected the app to shutdown correctly

**Screenshots**
![Screen Shot 2020-05-03 at 09 12 07](https://user-images.githubusercontent.com/3316532/80915277-3e7b6c00-8d1f-11ea-85b8-7a273d8fd65a.png)

**Desktop (please complete the following information):**
 - OS: Linux

**Comment by lrettig** [comment]
Full error text:

```Unhandled Promise Rejection
TypeError: Cannot read property 'pid' of undefined
    at n (/opt/Spacemesh/resources/app.asar/desktop/main.prod.js:2:17894)```

Possibly related to #501 and #502, since all three issues seem to be occurring together.

**Comment by IlyaVi** [comment]
fixed in #495 
bug was caused by trying to kill node process before it actually started
